go/vt/sqlannotation/sqlannotation.go

```diff
@@ -25,6 +27,10 @@ const (
 	deleteDML                            = "delete"
 )
 
+var (
+	replicationUnfriendlyCountStat = stats.NewInt("replication-unfriendly-statements-count")
```

Comment:
I believe open-source convention is that we use UpperCamelCase. We do break it a couple of times - and I realize the example I pointed you to was wrong, sorry!